client/views/voip/VoIPLayout.tsx

```diff
@@ -213,6 +270,7 @@ class VoIPLayout
 			this.callTypeSelection.current.disabled = false;
 		}
 		this.userHandler?.unregister();
+		this.deInitUserAgent();
```

Comment:
instead of `deInit`, can we use something more descriptive, like `unregisterAgent`? 👀 (or `resetAgent`)

Reply:
Unregister confuses with SIP verb REGISTER. I will go for resetAgent